src/main/java/sirius/kernel/commons/Value.java

```diff
@@ -893,11 +887,8 @@ public Integer getInteger() {
             if (data instanceof Integer integer) {
```

Comment:
Left this as this is exactly the return type. Ok?

Reply:
For simplicity sake I would include it in the Number case and remove this if. When looking at the Implementation of the Integer or Long class they do exactly the same